docs/services/vision/classification.md

```diff
@@ -0,0 +1,216 @@
+            "label_path": "/path/to/labels.txt",
+            "num_threads": 1
+        }
+    }
+ ]
+}
+```
+
+Click **SAVE CONFIG** and head to the **COMPONENTS** tab.
+
+{{%expand "You can also configure the entire Vision Service and classifier in raw JSON" %}}
+
+To add a vision model to your robot, add the `name`, `type`, and `parameters` of the desired classifier to the `register_models` in the attributes field of the Vision Service config.
+For example:
+
+``` json {class="line-numbers linkable-line-numbers"}
+"services": [
+    {
+        "name": "vision1",
+        "type": "vision",
+        "attributes": {
+          "register_models": [
+            {
+              "name": "my_color_detector",
+              "type": "color_detector",
+              "parameters": {
+                "detect_color" : "#A3E2FF",
+                "hue_tolerance_pct": 0.06,
+                "segment_size_px": 100
+              }
+            },
+            {
+              "name": "my_classifier",
+              "type": "tflite_classifier",
+              "parameters": {
+                "model_path" : "/path/to/model.tflite",
+                "label_path": "/path/to/labels.txt",
+                "num_threads": 1
+              }
+            },
+            {
+                "name": "my_segmenter",
+                "type": "radius_clustering_segmenter",
+                "parameters": {
+                    "min_points_in_plane": 2,
+                    "min_points_in_segment": 2,
+                    "clustering_radius_mm": 3.0,
+                    "mean_k_filtering": 0
+                }
+            }
+          ]
+        }
+    }
+]
+```
+
+{{% /expand%}}
+
+### Add a camera component and a "transform" model
+
+You cannot interact directly with the [Vision Service](/services/vision/).
+To be able to interact with the Vision Service you must:
+
+1. Configure a physical [camera component](../../../components/camera) to wrap the service.
+2. Configure a [transform camera](../../../components/camera/transform) to view output from the classifier overlaid on images from the physical camera.
+
+After adding the component and its attributes, click **SAVE CONFIG**.
+
+Wait for the robot to reload, and then go to the **CONTROL** tab to test the stream of detections.
+
+## Code
+
+The following code gets the robot’s Vision Service and then runs a classifier vision model on an image from the robot's camera named `"camera_1"` in this example.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.services.vision import VisionServiceClient, VisModelConfig, VisModelType
+
+robot = await connect()
+# grab camera from the robot
+cam1 = Camera.from_robot(robot, "cam1")
+# grab Viam's vision service which has the classifier already registered
+vision = VisionServiceClient.from_robot(robot)
+
+print("Vision Resources:")
+print(await vision.get_detector_names())
```

Comment:
This should say "get_classifier_names()"

Reply:
```suggestion
print(await vision.get_classifier_names())
```